jscomp/main/bsb_main.ml

```diff
@@ -160,14 +160,17 @@ let install_target config_opt =
 
 (* see discussion #929, if we catch the exception, we don't have stacktrace... *)
 let () =
+  let getenv_opt env = try Some(Sys.getenv env)
```

Comment:
I'd love it if we could make this a command line arg, like `-build-artifacts-dir` that esy would automatically pass. It would make the flags reusable for other purposes, and usable from the command line.

Reply:
I agree, the current solution is mostly to move quickly, but I can add this flag now as well 👍